client/src/Main.elm

```diff
@@ -257,85 +272,104 @@ view model =
 catalogueView : Model -> Element Msg
 catalogueView model =
     column
-        [ Element.width Element.fill
-        , Element.height Element.fill
-        , spacing 25
+        [ spacing 25
         , padding 50
-        , centerX
         ]
         [ title "Vector Catalog"
         , row [ spacing 20 ]
             [ button_ (Just ToggleAll) "Toggle all"
             ]
-        , Accordion.accordion
```

Comment:
Revert this please. Style the `Accordion` in `Accordion.elm`

Reply:
Done!